src/ts/waterfall/details-overlay/extract-details-keys.ts

```diff
@@ -24,6 +32,10 @@ function formatMilliseconds(millis: number): string {
   return `${millis} ms`;
 }
 
+function formatDateLocalized(d: Date): string {
+  return `${d.toUTCString()}</br>(local time: ${d.toLocaleString()})`;
```

Comment:
Could this cause issues when the recording machine is in a different (non GMT) time-zone than the user looking at the graph? There might be a danger of hiding correlations to backend issues.